src/helix/endpoints/channels/start_commercial.rs

```diff
@@ -48,24 +48,25 @@
 //!
 //! You can also get the [`http::Request`] with [`request.create_request(body, &token, &client_id)`](helix::RequestPost::create_request)
 //! and parse the [`http::Response`] with [`StartCommercialRequest::parse_response(None, &request.get_uri(), response)`](StartCommercialRequest::parse_response)
+use std::marker::PhantomData;
+
 use super::*;
 use helix::RequestPost;
 
 // Not implementing builder since it's not really needed...
 /// Query Parameters for [Start Commercial](super::start_commercial)
 ///
 /// [`start-commercial`](https://dev.twitch.tv/docs/api/reference#start-commercial)
-#[derive(PartialEq, Eq, Deserialize, Serialize, Clone, Debug)]
+#[derive(PartialEq, Eq, Deserialize, Serialize, Clone, Debug, Default)]
 #[non_exhaustive]
-pub struct StartCommercialRequest {}
-
-impl StartCommercialRequest {
-    /// Create a new [`StartCommercialRequest`]
-    pub fn new() -> Self { StartCommercialRequest {} }
+pub struct StartCommercialRequest<'a> {
+    #[serde(skip)]
+    _marker: PhantomData<&'a ()>,
```

Comment:
is this really needed?

Reply:
This is needed, because of the later impl of `RequestPost`:
```rust
impl<'a> RequestPost for StartCommercialRequest<'a> {
    type Body = StartCommercialBody<'a>; // <- cannot use '_
}
```

Reply:
ugh, this is why `GAT` is nice